Gemfile

```diff
@@ -36,7 +38,7 @@ gem 'sinatra', '>= 3.0.6'
 gem 'ssh_data'
 gem 'travis'
 gem 'travis-config', github: 'travis-ci/travis-config'
-gem 'travis-github_apps', github: 'travis-ci/travis-github_apps'
+gem 'travis-github_apps', git: 'https://github.com/travis-ci/travis-github_apps', branch: 'ga-ext_access'
```

Comment:
Why are we downgrading here? This gem had some recent updates but we are moving back to some 4 years old branch.